codecov.yml

```diff
@@ -2,7 +2,7 @@ coverage:
   status:
     project:
       default:
-        threshold: 0.2
+        threshold: 20%
```

Comment:
What? 20%??

Reply:
We crazy or what?

Reply:
Lol. I did it to test whether "0.2" and "20%" are the same thing (cos at the time the CI was failing - so if it passed with "20%" that means they're not the same thing). Clearly they're not the same thing!

I'll change it to 0% soon and only include the "lib.rs" of our custom pallets